nengo_loihi/tests/test_validate.py

```diff
@@ -16,7 +16,7 @@ def test_validate_block():
     # too many input axons
     block = LoihiBlock(410)
     block.add_synapse(Synapse(5000))
-    with pytest.raises(BuildError, match="[Ii]nput axon"):
```

Comment:
Did codespell start complaining about these? That's kind of annoying, because I did this intentionally in case we change the warning slightly (e.g. to say something like "The input axons" instead of "Input axons"). It's not a big deal, because we can just change the tests if the warning changes, but I am annoyed that codespell complains about this, so I just needed somewhere to complain.

Reply:
I think it was just `utput` that it complained about, figured I might as well make it consistent for all of them. It would be nice if `pytest.raises` could be case insensitive too, but doesn't seem like that's possible right now